src/reactionsystem.jl

```diff
@@ -500,7 +500,7 @@ struct ReactionSystem{V <: NetworkProperties} <:
     """Dependent unknown variables representing species"""
     species::Vector{BasicSymbolic{Real}}
     """Parameter variables. Must not contain the independent variable."""
-    ps::Vector{BasicSymbolic{Real}}
+    ps::Vector{Any}
```

Comment:
Maybe also add a check in the constructor that every element is a `BasicSymbolic` with an error if they aren't?

Reply:
That's a good point, yes, will do this.

Reply:
Added such a sheck